examples/sample_dynamic_requires.py

```diff
@@ -1,7 +1,7 @@
 import luigi
 
-import gokart
-'''
+import gokart  # noqa isort:skip
```

Comment:
[ASK] Why noqa?

Reply:
++

Reply:
@vaaaaanquish 
This row causes a conflict between yapf and isort.
- yapf wants no new line between this line and next multiple rows document
- isort wants new line between them

Do you know any setting to solve this problem?

Reply:
I've managed the above conflict by setting yapf `BLANK_LINES_BETWEEN_TOP_LEVEL_IMPORTS_AND_VARIABLES ` option.

Reply:
LGTM

Reply:
👍

Reply:
@mski-iksm 

> I've managed the above conflict by setting yapf BLANK_LINES_BETWEEN_TOP_LEVEL_IMPORTS_AND_VARIABLES option.

You should add this config on yapf.ini

Reply:
@hirosassa @vaaaaanquish 
> I've managed the above conflict by setting yapf BLANK_LINES_BETWEEN_TOP_LEVEL_IMPORTS_AND_VARIABLES option.

This actually couldn't solved the conflict. (Since this option is only implemented at the master branch of yapf and not released yet, yapf was not working in my developing environment, which I thought the conflict was solved.)

I couldn't find the option to solve this conflict, so instead I just moved the multiline document to above of imports.

Reply:
@mski-iksm Thank you for your dedicated investigation! I think this is OK in this PR.

Reply:
ref: related PR and last release is April 2020.
https://github.com/google/yapf/pull/854

Reply:
@hirosassa 
I've tested `BLANK_LINES_BETWEEN_TOP_LEVEL_IMPORTS_AND_VARIABLES` using the yapf mater branch code, but it only controls the space lines between import and variable definition (not docstring).


In the Google style, it seems that docstrings should be located at the top of the file. Maybe this is the reason why yapf doesn't support docstring located after import.
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html